2020/27xxx/CVE-2020-27858.json

```diff
@@ -1,18 +1,67 @@
 {
```

Comment:
CNA(s) of user (zdi) do not match owning CNA.

Reply:
We don't know how to remedy this. Our username is zdi-team, which we've used before, and we are the "CNA of last resort" for vulnerabilities that come through the ZDI program as listed here: https://cve.mitre.org/cve/request_id.html#cna_participants